plugins/woocommerce/src/Internal/Features/FeaturesController.php

```diff
@@ -441,6 +441,26 @@ private function init_feature_definitions(): void {
 				'is_legacy'          => true,
 				'enabled_by_default' => false,
 			),
+			'point_of_sale'          => array(
+				'name'               => __( 'Point of Sale', 'woocommerce' ),
+				'description'        => __(
+					'Enable Point of Sale functionality in the WooCommerce mobile apps.',
+					'woocommerce'
+				),
+				'enabled_by_default' => true,
+				'disable_ui'         => false,
+
+				/*
+				* This is not truly a legacy feature (it is not a feature that pre-dates the FeaturesController),
```

Comment:
❤️ Thanks for this comment.  I think the naming for `is_legacy` is really misleading and many new features set this to `true`.  Something I'd love for us to address in the future.